### DISCUSSION TITLE
ggplotly tidak dapat digunakan

### DISCUSSION THREAD

**Discussion by DeanAlexander27** [teamalgoritma/community#543] [Q&A] [answered]
Selamat Siang Kaka instruktur,
saya mengalami kendala dalam ggplotly untuk grafik pie chart dapat terlihat interactive.
dari library semua sudah tercantum dengan baik, dan untuk bar chart nya juga sudah bisa berjalan baik juga, hanya pie chart saja
tetapi kenapa masih menemukan kendala error ?
terima kasih 
![image](https://user-images.githubusercontent.com/123348110/218925562-d80f4781-8e43-4b3f-b0b9-c8cd6766e5d2.png)

**Comment by kevbow** [comment, marked as answer]
Dear @DeanAlexander27,

Hasil dari pie chart dari ggplot memang tidak dapat ditampilkan menggunakan ggplotly. 
Untuk errornya, apakah sudah dijalankan chunk tersebut secara keseluruhan?
Untuk kasus dari statement saya yang pertama, berikut saya lampirkan pertanyaan di stackoverflow:
- https://stackoverflow.com/questions/63737923/r-convert-ggplot-piechart-into-plotly

Demikian yang dapat saya sampaikan, semoga membantu.

**Reply by DeanAlexander27** [reply]
Dear Ka @kevbow 
Untuk errornya, apakah sudah dijalankan chunk tersebut secara keseluruhan? ->Sudah ka..
setelah mencari di beberapa source sudah ketemu ka permasalahan ini
https://plotly.com/r/pie-charts/

terima kasih ka Kevin..
